src/shapes/itext.class.ts

```diff
@@ -283,6 +283,18 @@ export class IText extends ITextClickBehaviorMixin<ITextEvents> {
     this.renderCursorOrSelection();
   }
 
+  /**
+   * @override block cursor/selection logic while rendering the exported canvas
+   * @todo this workaround should be replaced with a more robust solution
```

Comment:
i made years ago a renderExport function for those cases.
I would run the usual render code but without side effects and without runing the current caches and with less blur.
I should officialize it, it covers also those cases.

Reply:
This is fine anyway

Reply:
I am for. An object should have a method for that.
BTW #8298 offers a solution as well.